.vsts-dotnet-ci.yml

```diff
@@ -142,6 +152,7 @@ phases:
 
 #- phase: MonoOnMac
 #  queue: 'Hosted macOS Preview'
+#  timeoutInMinutes: 60
```

Comment:
nit: I know it's just a comment, but the spacing would be wrong if this were uncommented.  `timeoutInMinutes` should be a child of `queue`

Reply:
Thanks! I'm deleting the commented-out ones in my PR to master; there were several things like this that had drifted.